src/main/scala/wdl4s/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -276,6 +283,7 @@ class WdlStandardLibraryFunctionsType extends WdlFunctions[WdlType] {
   def write_json(params: Seq[Try[WdlType]]): Try[WdlType] = Success(WdlFileType)
   def glob(params: Seq[Try[WdlType]]): Try[WdlType] = Success(WdlArrayType(WdlFileType))
   def size(params: Seq[Try[WdlType]]): Try[WdlType] = Success(WdlFloatType)
+  def length(params: Seq[Try[WdlType]]): Try[WdlType] = Success(WdlIntegerType)
```

Comment:
Some of them are. But on investigation those are the ones which give different result types depending on the input so this might be ok...